Sources/Layers/Concat.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Concat.swift
+//  Bender
+//
+//  Created by Diego Ernst on 6/27/17.
+//
+//
+
+import MetalPerformanceShaders
+
+open class Concat: NetworkLayer {
+
+    var pipeline: MTLComputePipelineState!
+    let axis: LayerSizeAxis
+
+    let maxInputTextures = 10 // must match kernel max input textures
```

Comment:
if this is something that needs to be set, maybe would be better to make it a `var` or receiving it in the constructor if possible.